source/sass/11-base/toolbar/_toolbar.scss

```
@@ -1,36 +1,109 @@
-// Restore fonts, margins and paddings in admin toolbar.
+// Restore fonts, margins and paddings in adminimal admin toolbar.
 body {
-  // Fix adminimal toolbar font size issue.
-  .toolbar {
-    html, body, body div, span, object, iframe, h1, h2, h3, h4, h5, h6, p, blockquote, pre, abbr, address, cite, code, del, dfn, em, img, ins, kbd, q, samp, small, strong, sub, sup, var, b, i, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, caption, tbody, tfoot, thead, tr, th, td, article, aside, figure, footer, header, hgroup, menu, nav, section, time, mark, audio, video {
-      font-size: 100%;
-    }
+  .adminimal-admin-toolbar {
+    // Fix adminimal toolbar font size issue.
+    .toolbar {
+      html,
+      body,
+      body div,
+      span,
+      object,
+      iframe,
+      h1,
+      h2,
+      h3,
+      h4,
+      h5,
+      h6,
+      p,
+      blockquote,
+      pre,
+      abbr,
+      address,
+      cite,
+      code,
+      del,
+      dfn,
+      em,
+      img,
+      ins,
+      kbd,
+      q,
+      samp,
+      small,
+      strong,
+      sub,
+      sup,
+      var,
+      b,
+      i,
+      dl,
+      dt,
+      dd,
+      ol,
+      ul,
+      li,
+      fieldset,
+      form,
+      label,
+      legend,
+      table,
+      caption,
+      tbody,
+      tfoot,
+      thead,
+      tr,
+      th,
+      td,
+      article,
+      aside,
+      figure,
+      footer,
+      header,
+      hgroup,
+      menu,
+      nav,
+      section,
+      time,
+      mark,
+      audio,
+      video {
+        font-size: 100%;
+      }
 
-    a {
-      font-size: .65rem !important;
+      a {
+        font-size: 0.65rem !important;
+      }
     }
-
-    ul {
-      li {
-        margin-bottom: 0;
+  }
+  &.gin--vertical-toolbar {
+    .toolbar {
+      &-menu {
+        .menu-item {
+          .toolbar-menu {
+            .menu-item {
+              a.toolbar-icon {
+                font-size: 0.8rem;
+              }
+            }
+          }
+        }
       }
     }
   }
-
   // Fix adminimal toolbar icon issue.
   .toolbar-icon {
     font-family: inherit;
   }
 }
 
 // Fix fonts on logged in pages.
-body[class*='icon-'] {
+body[class*="icon-"] {
   font-family: inherit;
-  font-size: .9rem;
   font-style: inherit;
   font-variant: inherit;
   font-weight: inherit;
   text-decoration: inherit;
   text-transform: inherit;
   speak: inherit;
-}
+}
```

Missing end of file newline.
```suggestion
}

```